packages/core/bootstrap/src/lib/external-adapter/requester.ts

```diff
@@ -105,15 +104,15 @@ export class Requester {
     result?: number,
     results?: { [fsym: string]: number },
   ): AxiosResponseWithLiftedResult<T> | AxiosResponseWithPayloadAndLiftedResult<T> {
-    const isObj = type(response.data) === '[object Object]'
+    const isObj = deepType(response.data) === 'object'
     const output = isObj
       ? (response as AxiosResponseWithLiftedResult<T>)
       : ({
           ...response,
           data: { payload: response.data },
         } as AxiosResponseWithPayloadAndLiftedResult<T>)
-    output.data.result &&= result
-    output.data.results &&= results
+    if (result) output.data.result = result
```

Comment:
The &&= syntax wasn't working for me. It checks the left side `output.data.result`, when it needs to be checking `result`